install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -1,4 +1,5 @@
-# {{ if or .Values.global.crds (semverCompare ">=2.10.0-0" .Capabilities.TillerVersion.SemVer) }}
```

Comment:
I strongly prefer we keep in place for 1.0 the ability to run install with one operation with this existing conditional.

Otherwise, once other comments are addressed, looks good.